Index: appEntrega/models.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from django.db import models\r\n\r\n\r\nclass Cliente(models.Model):\r\n    cif = models.CharField(max_length=12, primary_key=True)\r\n    nombre_empresa = models.CharField(max_length=40)\r\n    email = models.EmailField(max_length=254)\r\n    telefono = models.IntegerField(max_length=9, unique=True)\r\n    nombre_cliente = models.CharField(max_length=20)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/appEntrega/models.py b/appEntrega/models.py
--- a/appEntrega/models.py	(revision 7ac08136b8d6b5836eb678ca86449bf12a164929)
+++ b/appEntrega/models.py	(date 1619189169522)
@@ -7,3 +7,6 @@
     email = models.EmailField(max_length=254)
     telefono = models.IntegerField(max_length=9, unique=True)
     nombre_cliente = models.CharField(max_length=20)
+
+
+
